### PR TITLE
chore(cli): Use large runners

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

**Comment by hermanschaaf** [comment]
@yevgenypats what's the summary / reasoning behind the change here?

**Comment by yevgenypats** [comment]
> @yevgenypats what's the summary / reasoning behind the change here?

Im still playing with it but I think we should upgrade our CI to use large runners where possible. they are quite slow now for now reason and I think we can pay for the increased speed

File: .github/workflows/cli.yml [hunk 1]
```diff
@@ -87,9 +67,9 @@ jobs:
           path: |
             ~/.cache/go-build
             ~/go/pkg/mod
-          key: ${{ runner.os }}-go-1.19.5-release-cache-${{ hashFiles('cli/go.sum') }}
+          key: ubuntu-go-1.19.5-release-cache-${{ hashFiles('cli/go.sum') }}
           restore-keys: |
-            ${{ runner.os }}-go-1.19.5-release-cache-cli
+            ubuntu-go-1.19.5-release-cache-cli
```
Comment (hermanschaaf) [inline .github/workflows/cli.yml:72]: Shouldn't this be using `${{ matrix.os }}`?

Comment (yevgenypats) [reply .github/workflows/cli.yml:72]: Im not sure this is needed anymore as this step doesn't run on matrix.

Comment (hermanschaaf) [reply .github/workflows/cli.yml:72]: Ah I see, gotcha

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**